WinApi/Windows/NativeWindow.cs

```diff
@@ -43,7 +43,7 @@ public string GetText()
             if (size > 0)
             {
                 var sb = new StringBuilder(size);
```

Comment:
This also has to be size + 1. 
Simply declaring a new variable `var length = size +1` and using them would ideal.